_includes/v20.1/app/insecure/basic-sample-pgx.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/jackc/pgx/v4"
+)
+
+func main() {
+	config, err := pgx.ParseConfig("postgresql://maxroach@localhost:26257/bank?sslmode=disable")
+	if err != nil {
+		log.Fatal("error configuring the database: ", err)
+	}
+
+	config.TLSConfig.ServerName = "localhost"
```

Comment:
well actually, I wouldn't expect ServerName to be required here. does this fail without specifying `ServerName`?

Reply:
You're right. In the insecure example, it doesn't fail. I'll remove that line from the insecure example.